NBitcoin/Transaction.cs

```diff
@@ -994,7 +994,14 @@ public WitScript(string script)
 			_Pushes = new byte[parts.Length][];
 			for (int i = 0; i < parts.Length; i++)
 			{
-				_Pushes[i] = Encoders.Hex.DecodeData(parts[i]);
+				if (parts[i] == "0")
+				{
+					_Pushes[i] = new byte[0];

```

Comment:
Don't do this. the `0` syntax is just short for `OP_0` which is only valid for a `Script`, not a `witScript`, because witScript should not have any script instructions.